src/routes/productpage/ProductPage.jsx

```diff
@@ -0,0 +1,58 @@
+import "./product-page.css";
+import { Filter, ProductCard } from "../../components";
+import { useProduct } from "../../context/product-context";
+import { useReducer } from "react";
+import { useEffect, useState } from "react";
+import { filterProducts } from "../../utils/filter-products";
```

Comment:
Separate the local imports with module imports will increase readability.
Example:
//All module inputs
import { useReducer } from "react";

//All local imports
import { useProduct } from "../../context/product-context";